### PR TITLE
Updated readme & added code of conduct

### PR DESCRIPTION
**Pull request by ctbarrett**
Based off the DevOps Days code of conduct. Not that we need it, necessarily, but I figured it wouldn't hurt (same thought process with the Readme updates).

All feedback gratefully appreciated!

### CODE REVIEW

File: CodeOfConduct.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+CoffeeOps event organizers.
+
+1. If I am offended or harassed, I will inform people around me who make me
+feel safe and the event organizers. If I feel safe, at my discretion, I will
+inform those giving offense of the specific actions with the hope that the
+other party is well-intentioned and ignorant, but I am under no obligation to
+do so.
+
+1. I understand that people are different and I attempt to be forgiving of
+others actions at the level of their sincere intent, but my priority is
```
Comment (frightenedmonkey) [inline CodeOfConduct.md]: Grammar nit: there's a bit of confusion as to the plural possessive of `others`. I *think* that `other's actions` would be acceptable, but the way to shortcut the issue would be to change this to `forgiving of one another's actions at the level of their sincere intent`.

File: CodeOfConduct.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+1. I am coming to CoffeeOps to interact with people. I understand that imagery
+and language which is suggestive or derogatory will offend and make people
+uncomfortable. I also understand that people may have boundaries and
+sensibilities different from my own. I will accept without question when
+informed that something is offensive or unacceptable in the context of the
+CoffeeOps event.
+
+1. I will never intentionally harass or offend another attendee regardless of
+gender, sexual orientation, disability, appearance, size, race or religion and
```
Comment (frightenedmonkey) [inline CodeOfConduct.md]: Grammar nit: I'd add a semicolon after religion, and change the `and` to an `I` so it reads: `race or religion; I will not abide`.

File: CodeOfConduct.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+# Seattle CoffeeOps - Code Of Conduct
+
+## ANTI-HARASSMENT POLICY
+
+CoffeeOps is dedicated to providing a harassment-free
+experience for everyone, regardless of gender, sexual orientation,
+disability, physical appearance, body size, race, or religion. We do not
```
Comment (frightenedmonkey) [inline CodeOfConduct.md:7]: You use the oxford comma here, and in one other spot below, but not over the entire document. I'd settle on one, just for consistency.

**Comment by frightenedmonkey** [comment]
This looks good to me, aside from my junky grammar nits. I like the idea of having a CoC, and the content of it appears to line up well to our general expectations. I am definitely in favor of this.

**Comment by ctbarrett** [comment]
Thanks for the feedback, @frightenedmonkey, I liked and agreed with all your suggestions. I _think_ I got all the commas consistent, but if you all could take another look, that would be awesome. Thanks!

**Comment by dougireton** [comment]
@ctbarrett thank you for this PR. I agree that it's past time for us to have a Code of Conduct.

After reading through the Code of Conduct in this PR, I'd rather adopt the Contributor Covenant Code of Conduct (making the appropriate changes to apply to a meet-up instead of an open source project).

[Contributor Covenant 1.4](http://contributor-covenant.org/version/1/4/)

I'm happy to make the changes and resubmit.

**Comment by ctbarrett** [comment]
@dougireton thanks! That looks great, as well - feel free to close and resubmit, or I can take a pass at it later this week and update the PR.

**Comment by dougireton** [comment]
Closing for now. I'll resubmit with the Contributor Covenant Code of Conduct.

**Pull request closed without merging**